docs/en/llm/proxy_server.md

```diff
@@ -30,6 +32,65 @@ APIs related to usage include:
 
 The usage of these APIs is the same as that of api_server.
 
+### add, delete and query through commands
+
+```shell
+curl -X 'GET' \
+  'http://localhost:10086/nodes/status' \
+  -H 'accept: application/json'
+```
+
+```shell
+curl -X 'POST' \
+  'http://localhost:10086/nodes/add' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "url": "http://0.0.0.0:23333"
+}'
+```
+
+```shell
+curl -X 'POST' \
+  'http://localhost:10086/nodes/remove?node_url=http://0.0.0.0:23333' \
```

Comment:
Which is a more conventional way, "remove/?node_url" or set the parameter in `-d` field?

Reply:
Both worked, the command was from swagger UI.